scribus/styles/paragraphstyle.h

```diff
@@ -148,14 +148,12 @@ class SCRIBUS_API ParagraphStyle : public Style
 	}
 #include "paragraphstyle.attrdefs.cxx"
 #undef ATTRDEF
-	
-	
+	CharStyle m_cstyle;
 private:
 		
 	// member declarations:
 	StyleContextProxy m_cstyleContext;
 	bool m_cstyleContextIsInh;
-	CharStyle m_cstyle;
```

Comment:
Why was this private member made public? If you really need it add an accessor function.